.github/workflows/release.yml

```diff
@@ -316,3 +316,21 @@ jobs:
             --tag ${{ env.DOCKER_IMAGE }}:${{ env.RELEASE_VERSION }} \
             ${{ inputs.isLatest && format('--tag {0}:latest', env.DOCKER_IMAGE) || '' }} \
             ${{ steps.build-docker.outputs.image }}
+  snyk:
+    name: Snyk Monitor
+    needs: [ docker, release ]
+    # skip if the version contains a dash as a quick test for -alpha, -rc, -SNAPSHOT, etc.
+    if: ${{ !contains(inputs.releaseVersion, '-') }}
```

Comment:
The idea is to skip alpha, snapshot, rc, etc., anything that isn't a production version. However, expressions don't support regex, so this is the simplest I came up with now. Ideas are welcome :+1: